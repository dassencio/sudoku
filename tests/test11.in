3 5
 _ 12  _ 10  _  _  _  9 13  4  _  _  _  _  _
 _  _  3  1  _  _  7 15  _  _ 10  _  9  4  _
 2  8  _  4  _  _  _ 14  1 10  _ 13 11 12  _
 7  9 14  _  1  _  _  8  _  _ 15  _  _  _  _
 _ 10  _ 12  _  _  _  _ 15  _  9  _ 14  _  _
 8 15  _  _  6 12  _ 10  _  _  _  _  _  _  5
10  6  _ 14 12  _  _  _  _ 11 13  _  8  _  _
 _  1  _  _  2  _  _  _  _  _ 11  _  _ 10  _
 _  _  8  _ 11  7  _  _  _  _  1  6  _ 14  2
14  _  _  _  _  _  _  6  _  2  8  _  _ 11  1
 _  _  4  _ 10  _ 14  _  _  _  _  5  _  9  _
 _  _  _  _  5  _  _  7  _  _  3  _ 10  6 14
 _ 14  5  3  _  6 10 11  _  _  _  2  _ 15  8
 _  4 12  _  8  _  _  3 14  _  _ 11  5  _  _
 _  _  _  _  _  1  8  5  _  _  _  3  _  7  _
