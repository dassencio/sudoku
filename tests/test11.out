15 12 11 10  7  8  6  9 13  4  5 14  2  1  3
13  5  3  1 14 11  7 15  2 12 10  8  9  4  6
 2  8  6  4  9  5  3 14  1 10  7 13 11 12 15
 7  9 14  5  1  4 13  8 11  3 15 10  6  2 12
 3 10 13 12  4  2  5  1 15  6  9  7 14  8 11
 8 15  2 11  6 12  9 10  7 14  4  1 13  3  5
10  6  7 14 12  9  1  2  3 11 13 15  8  5  4
 5  1 15 13  2 14 12  4  6  8 11  9  3 10  7
 4  3  8  9 11  7 15 13 10  5  1  6 12 14  2
14 13  9 15  3 10  4  6  5  2  8 12  7 11  1
11  7  4  6 10  3 14 12  8  1  2  5 15  9 13
12  2  1  8  5 13 11  7  9 15  3  4 10  6 14
 9 14  5  3 13  6 10 11  4  7 12  2  1 15  8
 1  4 12  7  8 15  2  3 14  9  6 11  5 13 10
 6 11 10  2 15  1  8  5 12 13 14  3  4  7  9
